hydrological-processes.csl

```diff
@@ -0,0 +1,220 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Hydrological Processes</title>
+    <id>http://www.zotero.org/styles/hydrological-processes</id>
+    <link href="http://www.zotero.org/styles/hydrological-processes" rel="self"/>
+    <link href="http://www.zotero.org/styles/aquatic-conservation" rel="template"/>
+    <link href="http://onlinelibrary.wiley.com/journal/10.1002/%28ISSN%291544-8452/homepage/ForAuthors.html" rel="documentation"/>
+    <author>
+      <name>Aditi Bhaskar</name>
+      <email>abhaskar@usgs.gov</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="geography"/>
+    <issn>0885-6087</issn>
+    <issn>1099-1085</issn>
+    <eissn>1099-1085</eissn>
```

Comment:
Yes. (eISSN stand for "electronic International Standard Serial Number")

You can update your pull request via the pencil (edit) button at https://github.com/aditibhaskar/styles/blob/patch-1/hydrological-processes.csl